huaweicloud/services/modelarts/resource_huaweicloud_modelarts_workspace.go

```diff
@@ -0,0 +1,452 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product ModelArts
+// ---------------------------------------------------------------
+
+package modelarts
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceModelartsWorkspace() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceModelartsWorkspaceCreate,
+		UpdateContext: resourceModelartsWorkspaceUpdate,
+		ReadContext:   resourceModelartsWorkspaceRead,
+		DeleteContext: resourceModelartsWorkspaceDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Workspace name, which consists of 4 to 64 characters.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```

Comment:
as we discussed before, the `description` should support to clear, so please delete this line and modify in Update func.

Reply:
ok,done